resources/schema.gql

```diff
@@ -138,24 +138,28 @@ type AccountMut {
 	"""
 	Update account email
 	"""
-	updateEmail(newEmail: String!): UpdateEmailResult!
+	updateEmail(newEmail: Email!): UpdateEmailResult!
 	"""
 	Create a new account
 	"""
-	createAccount(accountName: String!, email: String): CreateAccountResult!
+	createAccount(accountName: AccountName!, email: Email): CreateAccountResult!
 	"""
-	Reset password for selected account. Allowed only for admin users
+	Reset password for a selected account. Allowed only for admin users
 	"""
 	modifyPassword(accountName: AccountName!, password: String!): ModifyPasswordResult!
 	"""
+	Deleting an account by name
+	"""
+	deleteAccountByName(accountName: AccountName!): DeleteAccountResult!
+	"""
 	Access to the mutable flow configurations of this account
```

Comment:
I think our API got mad a bit. It contains collection-level operations (`createAccount`, `deleteAccountByName`) and individual-level operations (`updateEmail`, `modifyPassword`) at the same time.

I think `createAccount` should be a part of "type Accounts". `deleteAccountByName` is doubtful, maybe have `delete` at `Account` level, and just use `Accounts.byName` to locate it.

Reply:
Yes, fully agree -- will be updated